### PR TITLE
Update syntax for second parameter in split() method

### LINKED ISSUE #3402: [Edit] Python [Edit] Python Strings: Split
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

This edit is for the Python Strings: [Split](https://www.codecademy.com/resources/docs/python/strings/split) method located under the file path `docs/content/python/concepts/strings/terms/split/split.md`

Edit the syntax section to clarify that the second parameter in the `split()` method is the maximum number of splits. Also include the default when a number isn't given.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by christina-ml**
### Description

Closes #3402 

This edit is for the Python Strings: [Split](https://www.codecademy.com/resources/docs/python/strings/split) method located under the file path docs/content/python/concepts/strings/terms/split/split.md

Edit the syntax section to clarify that the second parameter in the split() method is the maximum number of splits. Also include the default when a number isn't given.

My changes:
- Edit syntax section to clarify that the second parameter in the split() method is the maximum number of splits
- Rename second parameter from `number_of_items` to `maxsplit` for clarification as to what this parameter does. It is not a count for the number of items, instead it sets a maximum number of times the string can be split.
- Explain what the default value is if second parameter is not specified, or if the second parameter has a value of `-1`

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/split/split.md [hunk 1]
```diff
@@ -21,13 +21,15 @@ The **`.split()`** method returns a new [list](https://www.codecademy.com/resour
...
 ```pseudo
-string.split(delimiter, number_of_items)
+string.split(delimiter, maxsplit)
 ```
 
 The `.split()` method takes the following optional parameters:
 
 - A `delimiter` that is either a [regular expression](https://www.codecademy.com/resources/docs/python/regex) or a string that is composed of one or more characters.
-- A maximum `number_of_items` for the returned list.
+- A maximum `maxsplit` specifying at most the number of splits that can occur, and the remainder of the string is returned as the final element of the list.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/split/split.md:30]:
```suggestion
- The value `maxsplit` specifies the total number of splits that can occur, and the remainder of the string is returned as the final element of the list. The default value is `-1`, which means an unlimited number of splits.
```

File: content/python/concepts/strings/terms/split/split.md [hunk 2]
```diff
@@ -21,13 +21,15 @@ The **`.split()`** method returns a new [list](https://www.codecademy.com/resour
...
+
+If the `maxsplit` parameter is not specified, or has a value of `-1`, the default value is `-1`. This means there is no maximum limit to the number of splits, and all possible splits are made.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/split/split.md:32]:
```suggestion
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@christina-ml Just a couple of edits. Please update the entry, and tag me when you've finished the changes. Thanks

**Comment by christina-ml** [comment]
@caupolicandiaz I updated the entry with the suggested changes.

**Review by caupolicandiaz — APPROVED** [review]
@christina-ml This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@christina-ml, @caupolicandiaz looks good to me.

**Pull request merged**